components/hab/win/chocolateyinstall.ps1

```diff
@@ -1,5 +1,5 @@
 $version = '$version$'
-$url = "https://bintray.com/habitat/stable/download_file?file_path=windows%2Fx86_64%2Fhab-$version-x86_64-windows.zip"
+$url = "https://packages.chef.io/files/stable/habitat/latest/hab-x86_64-windows.zip"

```

Comment:
won't this always get the latest? We want to make sure it is pinned to the version of the package so if someone runs `choco install habitat -version 0.88.0` they are guaranteed to get that version.

Reply:
That's a good question. Maybe my understanding of choco was off but I thought choco re-published our artifact.  If it's just a wrapper that tells the install how to fetch from upstream then yes, this will need to change.

Reply:
you can do either. We have been wrapping the upstream in order to keep the stats with bintray (likely with no value) however we may want to do the same with our own packages endpoint if that will help us to better track the stats. If not, then we could just embed the entire zip in the package.

Reply:
:+1:   I'll rework this to align with what we've been doing in the past.  We can re-evaluate later if we want to change how we bundle things, but I suspect we'll want the stats from package.chef.io

Reply:
@mwrock  With the latest changes: ```.expeditor/scripts/finish_release/publish_to_chocolatey.ps1
--- Fetching manifest
Fetching https://packages.chef.io/files/dev/habitat/latest/manifest.json
--- :thinking_face: Determining version and release to publish                                                                                             0.89.53                                                                                                                                                    --- Fetching checksum                                                                                                                                      https://packages.chef.io/files/dev/habitat/latest/hab-x86_64-windows.zip.sha256sum
5d1980a90f2153dd2133025efb19efea1120b76164083d81bbbe6916b4929c28
--- Patching nuspec
--- :bug: DEBUG PRINT PATCHED NUSPEC
$version = '0.89.53'
$url = "https://packages.chef.io/files/habitat/$version/hab-x86_64-windows.zip"
$unzipLocation = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
$checksum = '5d1980a90f2153dd2133025efb19efea1120b76164083d81bbbe6916b4929c28'

Install-ChocolateyZipPackage "habitat" $url $unzipLocation -checksum $checksum -checksumType sha256
--- Publishing package
--- NOT PUBLISHING: Build triggered by  and not Chef Expeditor
choco pack /tmp/4ktjdqng.drb/habitat.nuspec --version 0.89.53
choco push habitat.0.89.53.nupkg -k <elided ChocoApiKey> --timeout 600
PS /home/smacfarlane/src/habitat-sh/habitat> Invoke-RestMethod https://packages.chef.io/files/habitat/0.89.53/hab-x86_64-windows.zip.sha256sum
5d1980a90f2153dd2133025efb19efea1120b76164083d81bbbe6916b4929c28  hab-x86_64-windows.zip```

Reply:
thats cool. I wonder if you should comment out the `choco push` for now to make sure those don't get accidentally pushed to choco.

Reply:
Yup, the guard I have in place will only push if it's expeditor starting the job, so manual runs for verification are safe.   When we do have a real release, we'll want it to attempt to go all the way through, there's no good way to exercise that bit of code until the time comes though